pkg/cgroup/resolve_container.go

```diff
@@ -125,12 +125,9 @@ func getContainerInfo(cGroupID, pid uint64) (*ContainerInfo, error) {
 	// in the case the containerID is not a kubernetes container, the updateListPodCache will not add it to the cache
 	if _, ok := containerIDToContainerInfo[containerID]; !ok {
 		// some system process might have container ID, but we need to replace it if the container is not a kubernetes container
```

Comment:
this comment line not valid any more?

Reply:
this line is still valid

Reply:
setting `containerID` to `utils.SystemProcessName` doesn't add the system processes to the cache and prevent the next refresh. That's why we use `containerID` as it is in the cache.

Reply:
@rootfs I am wondering if having some system process with real containerID will impact system process metric aggregation in somehow.

Reply:
Maybe we need to update the containerID after adding it to the cache...

Sorry, I merged this PR, but we might need to update it...